test/cmdUtil.js

```diff
@@ -0,0 +1 @@
+require('../lib/cmdUtil.js').generateNewToken();
```

Comment:
Do we have any way to extend this test a little in a sensible way? This looks very bare-bones to me.